components/region_cache_memory_engine/src/engine.rs

```diff
@@ -73,22 +135,47 @@ pub struct RegionCacheMemoryEngine {
 }
 
 impl Debug for RegionCacheMemoryEngine {
-    fn fmt(&self, _: &mut fmt::Formatter<'_>) -> fmt::Result {
-        unimplemented!()
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "Region Cache Memory Engine")
     }
 }
 
 impl RegionCacheMemoryEngine {
-    pub fn new() -> Self {
-        RegionCacheMemoryEngine::default()
+    pub fn new_region(&self, region_id: u64) {
```

Comment:
how about rename it as "insert_region()"

Reply:
new maybe semantically suitable.